dgbpy/mlapply.py

```diff
@@ -376,10 +376,11 @@ def doTrain( examplefilenm, platform=dgbkeys.kerasplfnm, type=TrainType.New,
                                           nbchunks=params['nbchunk'],
                                           force=False,
                                           split=params['split'],nbfolds=params['nbfold'] )
-      if type != TrainType.New and dgbkeys.trainconfigdictstr in infos and dgbkeys.withunlabeleddictstr in dgbmlio.getInfo(examplefilenm):
+      hasUnlabels = dgbhdf5.hasUnlabeled(dgbmlio.getInfo(examplefilenm))
```

Comment:
You already have dgbmlio.getInfo(examplefilenm), this is the variable infos
Do not use capital letters in variable names

Reply:
infos from doTrain function is from modelin, not from examplefilenm, that's why I used new getInfo function.